Fix cucumber error introduced by full script in transaction output

philipr-za opened the pull request:
This PR updates the transaction builder in the Cucumber helpers to calculate the Beta hash and the Transaction Output hash correctly now that the whole script is included in the output.

## How Has This Been Tested?
PR updates tests

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
* [ ] I'm merging against the `development` branch.
* [x] I have squashed my commits into a single commit.

delta1 commented on integration_tests/helpers/transactionBuilder.js:
```diff
@@ -125,9 +125,10 @@ class TransactionBuilder {
 
     let beta = calculateBeta(
-      nopScriptHash,
+      nop_script_bytes,
```
I think we prefer `camelCase` for JS, and if lint doesn't complain then it should! Maybe I'm just bikeshedding, but everything else is camelCase (I think).

philipr-za replied:
fair enough

delta1 replied:
personally I prefer snake_case but 🤷‍♂️

philipr-za committed:
Fix cucumber error introduced by full script in transaction output

This PR updates the transaction builder in the Cucumber helpers to calculate the Beta hash and the Transaction Output hash correctly now that the whole script is included in the output.

Implement including full script in transaction outputs

Commit:
Merge branch 'tari-script' into philip-cucumber-full-script-fix
